pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go

```diff
@@ -62,6 +62,7 @@ var _ = Describe("VirtualMachineInstance Mutator", func() {
 	cpuModelFromConfig := "Haswell"
 	machineTypeFromConfig := "pc-q35-3.0"
 	cpuRequestFromConfig := "800m"
```

Comment:
nit: I think you can directly write `request.MustParse("800m")` and assign it instead of introducing a new variable.